index.html

```diff
@@ -282,8 +316,16 @@ <h2>About me</h2>
       </form>
 
       <div class="rectangle">
-        <img src="./images/projects/footericons/Rectangle 42.svg" alt="" />
+        <img
+          class="disappear"
+          src="./images/projects/footericons/Rectangle 42.svg"
+          alt="horizontal"
+        />
       </div>
+      <div class="crossimage1">
+        <img src="./images/desky/crossimage.svg" alt="crossimage" />
+      </div>
+
       <ul>
         <li>
           <a href="#"
```

Comment:
- [ ] As suggested by the previous reviewer, kindly consider removing this space from your footer below social icons in the footer as we need to make our design identical to the figma.
your design right now
![c](https://github.com/mj-e-boc/M-J-Portfolio/assets/96683843/f9499fc0-35da-4cf3-925b-1b53da75f8b2)

figma design

![f](https://github.com/mj-e-boc/M-J-Portfolio/assets/96683843/94164925-0b63-401d-bfe5-ce265a273d06)